File: Practice/08/C++/08.cpp
#include <iostream>
#include <string>
#include <vector>

using namespace std;

int main()
{
    string task;
    cout << "Input task: ";
    cin >> task;
    cout << endl;

    size_t sz;

    double a = stod(task, &sz);
    char opr = task.c_str()[sz];
    double b = stod(task.substr(sz + 1));

    switch (opr)
    {
    case '+':
        cout << task << " = " << a + b;
        break;
    case '-':
        cout << task << " = " << a - b;
        break;
    case '*':
        cout << task << " = " << a * b;
        break;
    case '/':
        cout << task << " = " << a / b;
        break;
    }
}

File: Practice/08/Python/08.py
operations = {
    '*': lambda a, b: a * b,
    '/': lambda a, b: a / b,
    '-': lambda a, b: a - b,
    '+': lambda a, b: a + b
}

a, opr, b = input("Введите операцию: ").split(" ")
a, b = int(a), int(b)
print(operations[opr](a, b))